app/ide-desktop/lib/dashboard/src/data/dataLinkSchema.json

```
@@ -100,7 +100,7 @@
         "libraryName": { "const": "Standard.AWS" },
         "uri": {
           "title": "URI",
-          "description": "Must start with \"s3://\".",
+          "description": "Must start with \"s3://\". Folder name must have at least 3 characters.",
```

```suggestion
          "description": "Must start with \"s3://\". Bucket name must have at least 3 characters.",
```

---

```suggestion
          "description": "Must start with \"s3://\". The bucket name must have between 3 and 63 characters.",
```

I guess that'd make sense if we want to be precise.

I'm not sure if this is needed at all - the bucket name must firstly be a valid S3 bucket. So the requirements for what is a valid name and what not are clearly defined when someone _creates_ a bucket.

---

good point, i'll just revert it then. definitely better to need less text